sdk/communication/Azure.Communication.Common/api/Azure.Communication.Common.netstandard2.0.cs

```diff
@@ -4,6 +4,9 @@ public partial class CallingApplicationIdentifier : Azure.Communication.Communic
     {
         public CallingApplicationIdentifier(string id) { }
         public string Id { get { throw null; } }
+        public override bool Equals(object obj) { throw null; }
```

Comment:
Why do we need to add those?

Reply:
It helps a lot with comparing if two identifiers are the same. Right now, we need to do a type check first and then check the values. I changed it so, `id1.Equals(id2)` would say if the point to the same Mri